crates/sui-types/src/error.rs

```diff
@@ -41,6 +41,19 @@ macro_rules! exit_main {
     };
 }
 
+#[macro_export]
+macro_rules! log_if_err {
+    ($e:expr, $($arg:tt)+) => {
+        {
+            let res = $e;
+            if let Err(e) = &res {
+                tracing::info!($($arg)+, "{}", e);
```

Comment:
Shouldn't this be `error` instead of `info`?

Reply:
We usually reserve error for internal errors, i.e. Things That Should Not Happen. But this macro can be used for errors that are about to be returned errors to clients, which should not have ERROR level.

Ideally we could specify the level in this macro but I'm trying not to spend my whole afternoon tweaking a macro 😂